### ISSUE TITLE
☂️  Flutter passes new revision of internal accessibility rating

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#75879]
This is an umbrella issue to keep track of the remaining works to pass our internal accessibility rating.
The new internal accessibility rating will be enforced by July 1, 2021

Web:

- [x] NVDA support https://github.com/flutter/flutter/issues/75880
- [ ] JAWS support https://github.com/flutter/flutter/issues/75881
- [x] iOS safari change font size https://github.com/flutter/flutter/issues/75882
- [x] Investigate high contrast mode https://github.com/flutter/flutter/issues/75883
- [x] tooltip meets accessibility standard https://github.com/flutter/flutter/issues/75885

mobile:
- [x] Switch control scrolling does not work https://github.com/flutter/flutter/issues/37974
- [x] Switch Access support https://github.com/flutter/flutter/issues/75886
- [x] Voice control support https://github.com/flutter/flutter/issues/75887
- [x] Voice access support https://github.com/flutter/flutter/issues/76827

**Comment by chunhtai** [comment]
@deckerst  I remember I have tested the scrolling, can you open a new issue for that with repro and link it here?

**Comment by deckerst** [comment]
@chunhtai here is the issue: https://github.com/flutter/flutter/issues/90912

**Comment by chunhtai** [comment]
closing this one since all the requirements are ether fixed or scheduled to be fixed
